helios/plato/django/server/utilityEndpoint.py

```diff
@@ -91,6 +91,9 @@ async def saveLayout(self, jsonData, returnValue):
                         returnValue["error"] = ""
                 except IntegrityError as e:
                     returnValue["error"] = f"not created, {str(e)}"
+                except:
+                        returnValue["result"] = "success"
+                        returnValue["error"] = ""
```

Comment:
Oh yeah, this is a bug fix to the Django server stuff.  Wondered where it went.  :laughing: